tests/tools/oesign/test-enclave/host/host.c

```diff
@@ -19,6 +39,19 @@ int main(int argc, const char* argv[])
         oe_put_err("Usage: %s enclave_image_path\n", argv[0]);
     }
 
+    /* Load the ELF image */
+    if ((result = oe_load_enclave_image(argv[1], &oeimage)) != OE_OK)
+    {
+        oe_put_err("oe_load_enclave_image(): result=%u", result);
+    }
+
+    /* Load the SGX enclave properties */
```

Comment:
Prior to this PR, this file is TEE-agnostic.  I think we should not change it to be single TEE only.  At minimum, I think this needs to test whether it's being signed for SGX before introducing SGX-only code into this code path.

Reply:
Related code move in block when KSS is enabled so setting will not applied if KSS not supported